### PR TITLE
Slight hover effect for table rows

### PR DESCRIPTION
**Pull request by pxlrbt**
As table rows are clickable, I think they would benefit from a slight hover effect:

https://user-images.githubusercontent.com/22632550/159869426-990cb598-3724-4e56-8282-7374cf506208.mov

### CODE REVIEW

**Comment by pxlrbt** [comment]
Yep, you're right. Fixed.

File: packages/tables/resources/views/components/row.blade.php [hunk 1]
```diff
@@ -1,3 +1,13 @@
-<tr {{ $attributes->class(['filament-tables-row']) }}>
+@props([
+    'recordUrl' => null,
+])
```
Comment (ryangjchandler) [inline packages/tables/resources/views/components/row.blade.php:3]:
```suggestion
])

```

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
Do we need to run the closure or just check it exists?

**Comment by danharrin** [comment]
I think you're right actually, ignore me

**Pull request merged**